ph5/core/ph5api.py

```diff
@@ -1454,111 +1454,59 @@ def get_availability(self, das, sample_rate, component,
         if not new_das_t:
             LOGGER.warning("No Das table found for " + das)
             return None
-        previous = {}
 
-        times = list()
+        gaps = 0
+        previous = {}
```

Comment:
This dict has a few keys that are over written, current are stored in `cur_...` this could be `prev_...` single variables